CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our project and community 
+a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, geographic location, or sexual
+identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
```

Comment:
```suggestion
As members of the Solid Community, we pledge to follow its [code of conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), which values are aligned with our own.

```
I think it's nice to reference the Solid Community code of conduct, but it may go in the attributions too.

Reply:
Maybe: 
```suggestion
We pledge to act and interact in ways that contribute to an open, welcoming,
diverse, inclusive, and healthy community. These values align with those of the 
[Solid Community's Code of Conduct]
(https://github.com/solid/process/blob/main/code-of-conduct.md).
```

Reply:
I wonder if the Solid org would be interested in adopting this CoC instead of a handwritten one.